File: gestion_compte/src/services/CompteService.java
package services;

import java.util.ArrayList;

import entity.Compte;

public class CompteService {
    private ArrayList<Compte> comptes=new ArrayList<>();
    public ArrayList<Compte> getComptes() {
        return comptes;
    }
    public CompteService() {
        
    }
    public boolean addCompte(Compte compte){
        comptes.add(compte);
        return true;
    }

    public Compte getCompteByNum(String numero){
      for (int index = 0; index < comptes.size(); index++) {
          if (comptes.get(index).getNumero().equals(numero)) {
              return  comptes.get(index);
          }
      }
        return null;
    }





    
}

File: php/src/service/CompteService.php
<?php 
require_once "./src/entity/Compte.php";
require_once "./src/repository/CompteRepository.php";
class CompteService{
   private CompteRepository $compteRepository;
   public function __construct()
   {
       $this->compteRepository =new CompteRepository();
   }

    public function addCompte(Compte $compte){
        $this->compteRepository->insert($compte);
    }

    /**
     * Get the value of comptes
     */
    public function getComptes(): array
    {
       return $this->compteRepository->selectAll();
    }

    
}